ezbake/config/conf.d/auth.conf

```diff
@@ -137,6 +137,15 @@ authorization: {
             name: "puppetlabs static file content"
         },
         {
+            match-request: {
+                path: "/puppet/v3/tasks"
+                type: path
```

Comment:
seems like locking this down to just GET might be good? I guess puppetserver will just 404 in the case of a PUT POST or DELETE so maybe it doesn't matter?